samples/excel/89-preview-apis/chart-series.yaml

```diff
@@ -0,0 +1,171 @@
+order: 1
+id: excel-89-preview-apis-chart-series-
+name: Chart series
+description: 'Add, set, and delete a series in a chart.'
+host: EXCEL
+api_set:
+    ExcelAPI: 1.7
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#add-series").click(() => tryCatch(addSeries));
+        $("#delete-series").click(() => tryCatch(deleteSeries));
+        $("#add-series-value").click(() => tryCatch(setSeriesValue));
+
+        async function addSeries() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Sample");
+
+                let seriesCollection = sheet.charts.getItemAt(0);
+                let rangeSelection = sheet.getRange("B1:B7");
```

Comment:
If I understand the "add series to a chart" scenario correctly, a user would want to add a series for a range of data that _does not already exist_ as a series in the chart.  In your snippet, the sample chart is created using data in the "Qtr1" column (B1:B7) -- so if you want to add another series to the chart, you should do so using data in another column (ex:  C2:C7).  

Suggestion:  
- change this line:  let rangeSelection = sheet.getRange("B1:B7");
- to this instead:  let rangeSelection = sheet.getRange("C2:C7");

Reply:
Done.